indicators/square-enix-ffxiv-gil-scam.yml

```diff
@@ -19,11 +19,9 @@ detection:
 
   loginPage:
     html|contains|all:
-      - '.php" method="post"'
+      - '.php"'
       - 'Square Enix ID'
       - 'Square Enix Password'
-      - 'One-Time Password'
-      - 'name="key_value"'
```

Comment:
This looks like it'll be too generic now? it'll flag on any PHP website containing those strings, not just this specific phishing kit?

Reply:
Updated the PR to make it more specific. It's unlikely a non-phishing login page would have all of those strings now.

All tests, positive and negative, in the first comment still pass.